storage/aptosdb/src/state_store/mod.rs

```diff
@@ -176,42 +176,42 @@ impl StateStore {
         Ok(result)
     }
 
-    pub fn get_value_at_version(
-        &self,
-        key_and_version: &(StateKey, Version),
-    ) -> Result<StateValue> {
+    /// Read the value for a specific version and require its existence. Call `get_value_by_value`
+    /// for most use cases.
+    fn get_value_at_version(&self, key_and_version: &(StateKey, Version)) -> Result<StateValue> {
```

Comment:
This name is confusing with `get_value_by_version` API, let's rename this to `get_value_updated_at_version` ?